### DISCUSSION TITLE
How to adjust width and height of gap between icons?

### DISCUSSION THREAD

**Discussion by tumashu** [lxqt/lxqt#2497] [Q&A]
I think the gap between icons seem to large, How to let X and Y small?

![图片](https://github.com/lxqt/lxqt/assets/67725/b2a6793e-38dc-4ad7-bbf3-a287425f6a54)

**Comment by stefonarch** [comment]
Check PCManFm-Qt → Preferences → Visualization → minimum space between icons.

**Comment by tumashu** [comment]
I only find 'Minimum item margins in icon view', but it seem to not for this feature

**Reply by stefonarch** [reply]
It's that, here it's like this with 0 x 0 :

![screen_area_lun_07:53:18_](https://github.com/lxqt/lxqt/assets/10681413/b89bcb60-eed6-4684-ab0f-8fb1139be727)

**Reply by tsujan** [reply]
Each item of the icon view has space for the icon, 13 characters, 3 lines and 4 pixels. The rest can be set by changing the above-mentioned setting.

**Comment by tumashu** [comment]
I set it to '0 x 0', but not get the result like your snapshot,  is it possible set to like "-10  x -10"
![Uploading 图片.png…]()

**Reply by tumashu** [reply]
![图片](https://github.com/lxqt/lxqt/assets/67725/6b454a39-d3f7-4704-9551-de7d3599673c)

**Comment by tsujan** [comment]
That's the impression you get from fonts, icons and texts. In this screenshot, the margin is 7🞩3, not 0🞩0, but the items are near each other:

![items](https://github.com/lxqt/lxqt/assets/981076/765f62f2-cb34-41ae-93b4-2562fda98587)

The reserved space also depends on how some fonts handle line heights and character widths. For example, Noto gives a relatively large height to each line.
